locator/tablets.cc

```diff
@@ -120,11 +130,28 @@ tablet_migration_streaming_info get_migration_streaming_info(const locator::topo
                 return !n || n->is_excluded();
             });
 
+            return result;
+        case tablet_transition_kind::repair:
+            auto s = std::unordered_set<tablet_replica>(tinfo.replicas.begin(), tinfo.replicas.end());
+            result.stream_weight = 2;
```

Comment:
let's use a symbol for the repair stream_weight instead of a constant, so if we decided to change the value we would do it in a single place.

Reply:
ok